node_modules/oae-authentication/tests/test-external-strategies.js

```diff
@@ -1341,14 +1286,7 @@ describe('Authentication', function() {
          * @param  {Function}   callback            Standard callback function
          */
         var _assertGoogleLogin = function(email, expectSuccess, callback) {
```

Comment:
We're leaving behind the pattern of the boolean flags for the `assert...Succeeds` and `assert...Fails`, so we can probably just convert this method into the 2 AuthenticationTestUtil methods.